test/chpldoc/classes/forwards.doc.chpl

```diff
@@ -0,0 +1,14 @@
+/* This class should get the forwarded methods from B */
```

Comment:
I'd consider making a version of this test where the field f is generic & calling the current output the .good (i.e. not a future).